core/src/main/kotlin/de/fraunhofer/iem/spha/core/strategy/WeightedMaximumKPICalculationStrategy.kt

```diff
@@ -24,7 +24,7 @@ internal object WeightedMaximumKPICalculationStrategy : BaseKpiCalculationStrate
 
     override fun internalCalculateKpi(edges: Collection<KpiHierarchyEdge>): KpiCalculationResult {
 
-        val max = if (edges.isEmpty()) 0 else edges.maxOf { it.to.score * it.actualWeight }.toInt()
+        val max = edges.maxByOrNull { it.to.score * it.actualWeight }?.to?.score ?: 0
```

Comment:
do we really want to have 0 as the fault value for a minimum strategy?

Reply:
updated to explicitly return the Empty()